### PR TITLE
Fix a CI script

### PR DESCRIPTION
**Pull request by aviflax**
As per commit comments.

### CODE REVIEW

File: tool/bin/install-sys-deps [hunk 1]
```diff
@@ -6,13 +6,13 @@
 
 set -eux
 
-OS="$1"
-
-if [ "$OS" == '' ]; then
+if [ "$*" == '' ]; then
```
Comment (mattsoutherden) [inline tool/bin/install-sys-deps]: This could be `if [ -z "$*" ]; then`

Comment (aviflax) [reply tool/bin/install-sys-deps]: Thanks! I couldn’t remember that operator. I’ll make this change. Thanks!

File: .circleci/config.yml [hunk 2]
```diff
@@ -45,7 +45,7 @@ jobs:
        - restore_cache:
            keys:
              - test-deps-v1-{{checksum "deps.edn"}}-{{checksum "bin/download-test-deps"}}
-       - run: bin/install-sys-deps << parameters.os >>
+       - run: ../.circleci/bin/install-sys-deps << parameters.os >>
```
Comment (miketheman) [inline .circleci/config.yml]: If this file is colocated with the config file, do you need to traverse down to step back up, or could you use 'bin/install...' instead?

Comment (miketheman) [reply .circleci/config.yml]: Oh, you were doing that already. So let me ask: why did you need to traverse directories? Add a comment to make it clear.

Comment (aviflax) [reply .circleci/config.yml]: Good point; this did in fact trip me up while working on this fix. The `..` traversal is due to the `working_directory` property of the job, which is just above the job’s steps, being set to `~/project/tool`.

I’ll add a comment to that effect before merging.

This is yet another reason to simplify the repo structure while implementing #226.

**Comment by aviflax** [comment]
> Since you're already renaming the file, would you consider adding a file extension like '.sh' for clarity?

I’m certainly open to it! Your question spurred me to try to do a little research on the pros and cons of using such an extension for shell scripts. For example, I read all the answers to [this question](https://unix.stackexchange.com/questions/31760/file-extensions-for-unix-shell-scripts). There are a _lot_ of opinions about this!

For me this boils down to the utility of being able to see at a glance that a script is a _shell_ script, vs the utility of being able to change the language of a script (e.g. to Python, Joker, etc) without having to change the name of the file and therefore changing its name everywhere it’s called. Right now I’m on the fence on this, so I’m going to sleep on it and see if my brain figures it out overnight

Thanks for the review!

**Comment by miketheman** [comment]
I found further support for no extension in the Google Style Guide https://google.github.io/styleguide/shell.xml#File_Extensions

**Comment by aviflax** [comment]
Good find Mike! Thanks for taking the time to look into this more. I think I’m inclined to leave the filename of the script as-is, without an extension. Can you live with that?

File: tool/bin/install-sys-deps (hunk 1 above)
Comment (aviflax) [reply tool/bin/install-sys-deps]: Done in 58ae250 — thanks!

**Comment by miketheman** [comment]
Yep, no extension makes sense. I am going to think about that style guide more for other projects, since the idea of an executable (binary, even though bash isn't a binary...) without an extension feels more natural to the user.

**Comment by mattsoutherden** [comment]
I concur. 👍

**Comment by aviflax** [comment]
OK, great. Now just waiting for someone to give me that sweet, sweet: ✅ before I merge.

**Pull request merged**